src/torch_ucc.cpp

```diff
@@ -60,7 +60,9 @@ const std::map<ReduceOp, ucc_reduction_op_t> ucc_op_map = {
     {ReduceOp::BAND, UCC_OP_BAND},
     {ReduceOp::BOR, UCC_OP_BOR},
     {ReduceOp::BXOR, UCC_OP_BXOR},
+#if TORCH_VERSION_MAJOR > 1 || (TORCH_VERSION_MAJOR == 1 && TORCH_VERSION_MINOR >= 11)
```

Comment:
from @pallab-zz: 
> Why do we need to take care of PT related checks here ? shouldn't these be in PT itself ?

@Sergei-Lebedev - My guess is because torch-ucc is an extension unlike NCCL PG, so this can be used by older version of Pytorch that doesn't define `ReduceOp::AVG`. Is my assumption correct? Do you see such case?

Reply:
@kingchc, yes, it's correct. Some of the our internal users prefer to use stable Pytorch 1.10 which doesn't have AVG op.